server/apis/v1/dexauthn.go

```diff
@@ -98,8 +98,8 @@ func (d *DexObject) oauth2Config(scopes []string) (*oauth2.Config, error) {
 	}, nil
 }
 
-func (d *DexObject) Authenticate(c *gin.Context) (authn.UserIdInfo, error) {
-	userInfo := authn.UserIdInfo{}
+func (d *DexObject) Authenticate(c *gin.Context) (authn.UserInfo, error) {
```

Comment:
BTW - the common practice is returning `(pointer, error)`, which makes more sense.

Reply:
+1.